.tours/4---innvocable apex actions.tour

```diff
@@ -0,0 +1,15 @@
+{
+    "title": "4 - Innvocable Actions",
+    "steps": [
+        {
+            "file": "es-space-mgmt/main/default/classes/ReservationManager.cls",
+            "description": "The ReservationManager apex class shows how to write apex actions for flows.",
+            "line": 2
+        },
+        {
+            "file": "es-space-mgmt/main/default/classes/ReservationManager.cls",
+            "description": "The Update Reservation Status method uses object agnostic design. Uses dynamic DML to update records. Admin can configure the fields that need an update in the custom metadata",
```

Comment:
This can be a part of the object agnostic tour.

Reply:
I think this is ok to have its own section. As the title says this is to show invocable apex actions than object agnostic design.